README.md

```diff
@@ -20,197 +20,170 @@
 <a href="https://slack.jina.ai"><img src="https://img.shields.io/badge/Slack-2.2k%2B-blueviolet?logo=slack&amp;logoColor=white"></a>
 </p>
 
-<!-- start elevator-pitch -->
+<!-- start ecosystem-description -->
 
 Jina is a neural search framework that empowers anyone to build SOTA and scalable deep learning search applications in minutes.
```

Comment:
```suggestion
Jina is a neural search framework that empowers anyone to build deep learning search applications in minutes.
```
Do we really need fancy words in the first sentence? Wouldn't it be better to keep it as simple as possible at the beginning and start adding complexity layers?

Reply:
neural search applications is the term we want to use and promote